cmd/chart-up/README.md

```diff
@@ -0,0 +1,47 @@
+# rke2-charts version references update
+
+Update rke2-charts version references with a single command.
+
+## Examples
+
+```sh
+# -c flag is not needed when CHARTS environment variable is present
+$ export CHARTS=$(mktemp -d '/tmp/XXXXXX')
+$ git clone -o upstream --depth 1 https://github.com/rancher/rke2-charts.git $CHARTS
+
+## print versions
+# chart-up <rke2 package name> [no args]
+$ chart-up rke2-canal-1.19-1.20
+appVersion: v3.13.3
+version: v3.13.3-build20211022
+rancher/hardened-calico: v3.13.3-build20210223
+rancher/hardened-flannel: v0.14.1-build20211022
+packageVersion: 5
+
+## print updated version, no file change
+# chart-up <rke2 package name> [field=version]
+$ chart-up rke2-canal-1.19-1.20 appVersion=v3.13.4
+appVersion: v3.13.4
+version: v3.13.3-build20211022
+rancher/hardened-calico: v3.13.3-build20210223
+rancher/hardened-flannel: v0.14.1-build20211022
+packageVersion: 5
+# Update multiple values including docker image version
+$ chart-up rke2-canal-1.19-1.20 rancher/hardened-calico=v3.13.5-build20220124 appVersion=v3.13.4
+appVersion: v3.13.4
+version: v3.13.3-build20211022
+rancher/hardened-calico: v3.13.5-build20220124
+rancher/hardened-flannel: v0.14.1-build20211022
+packageVersion: 5
```

Comment:
Oh, I didn't know that. I personally think the CI workflow should be aware of it and not the tool, but I might be wrong.

Reply:
The reason for that is that otherwise we would be overwriting an existing helm chart and users might get confused. The name of the chart contains the packageVersion: https://github.com/rancher/rke2-charts/tree/main/charts/rke2-canal/rke2-canal.
To be fully accurate, when changing the version, we should go back to `packageVersion: 1`

Reply:
I'll follow up on this concern on #34